data-plane/config/broker/500-receiver.yaml

```diff
@@ -114,13 +114,8 @@ spec:
             # This should be set according to initial delay seconds
             - name: WAIT_STARTUP_SECONDS
               value: "8"
-          command:
```

Comment:
Just curious, @pierDipi: why was this change needed for the introduction of ARM support

Reply:
that's how Jib works, it injects the entrypoint for the container, so there is no need to call java directly, also because the way Jib works is that it creates a directory that is the classpath with all the dependencies jar and the actual jar is separate, so the actual `java ...` command is a little more complicated than we had (and it wouldn't work downstream as well)